packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -700,7 +701,7 @@ export class SwapQuoter {
 
         if (
             opts.rfqt && // This is an RFQT-enabled API request
-            !opts.rfqt.isPriceAwareRFQEnabled && // If Price-aware RFQ is enabled, firm quotes are requested later on in the process.
+            !getPriceAwareRFQRolloutFlags(opts.rfqt.priceAwareRFQFlag).isFirmPriceAwareEnabled && // If Price-aware RFQ is enabled, firm quotes are requested later on in the process.
```

Comment:
Should this include a similar check for the indicative flag?

e.g !getPriceAwareRFQRolloutFlags(opts.rfqt.priceAwareRFQFlag).isIndicativePriceAwareEnabled &&

Reply:
no, because the firm and indicative quotes are 2 independent flags.

Reply:
indicative quotes are pulled here in the case of a false `isIndicativePriceAwareEnabled` flag: https://github.com/0xProject/protocol/blob/bf3035ad258f9a02581f7b5fca8ee8c6af047f37/packages/asset-swapper/src/utils/market_operation_utils/index.ts#L369

LGTM